src/main.rs

```diff
@@ -52,36 +53,41 @@ fn init() -> Result<(), anyhow::Error> {
         ));
     }
 
-    print_runtime_config(auth_token);
+    let url = match extract_url(&String::from_utf8_lossy(&result.stdout)) {
```

Comment:
Not sure the point of borrowing the temporary string here?

Reply:
Without the Borrow by making the input to the `extract_url` function a `String` requires adding a `.to_string()` which seems a tad bit odd? 

if that is more "rusty" - Happy to change to it.

Reply:
Do you not control the signature of the `extract_url` function?  I thought you did but I may have misunderstood...!

(If we want to be super "rusty" about it then the fancy schmancy idiom is to have `extract_url` take an `impl Into/AsRef/etc` so it can take a `String` or `&str` or `Cow<str>` and callers can use whichever is most convenient for them, but that's undoubtedly overkill in this case.)

Reply:
We do have control over the signature for `extract_url` but decided to leave those cleanups for later.